asyncpg/pool.py

```diff
@@ -409,6 +402,73 @@ def __init__(self, *connect_args,
         self._initialized = True
         return self
 
+    def set_connect_args(self, dsn=None, *,
+                         connection_class=connection.Connection, **kwargs):
```

Comment:
I wouldn't allow to change `connection_class` via `set_connect_args`, there's no clear use case for that.